fxmanifest.lua

```diff
@@ -2,20 +2,12 @@ fx_version 'cerulean'
 game 'gta5'
 
 description 'QB-Core'
-version '1.1.0'
+version '1.0.0'
 
 shared_scripts {
     'config.lua',
-    'shared/locale.lua',
-    'locale/en.lua',
-    'locale/*.lua',
-    'shared/main.lua',
-    'shared/items.lua',
-    'shared/jobs.lua',
-    'shared/vehicles.lua',
-    'shared/gangs.lua',
-    'shared/weapons.lua',
-    'shared/locations.lua'
+    'shared/*.lua',
+    'locale/*.lua'
```

Comment:
```suggestion
    'locale/en.lua',
    'locale/*.lua'
```
Let's add this back so languages can use English as a fallback.

Reply:
Btw, you know what I noticed when I did some tests using files in that way, it'll load the locale/en.lua twice because the *.lua means it will load every file no matter if it is loaded or not already